### ISSUE TITLE
Super Metroid Coop connected, but not sharing anything

### ISSUE THREAD

**Issue by GameCrazyMatt** [mcclure/emu-coop#32]
We're both using same files/emulators/scripts, and we each get an identical Coop mode: code when we begin our game, but taking damage from Ridley on ceres isn't sharing damage, nor does it enable my partner to leave on the elevator when the timer is engaged. Not receiving any errors. Lua script window is as following:

Playing Super Metroid
Connecting to server...
Logging in to server...
script returned but is still running registered functions
Searching for partner...
Connected to partner
Coop mode: (code here)

We've tried running the game with admin priv, to no avail.

**Comment by mcclure** [comment]
The emu-coop sharing code for Super Metroid does not share damage, and does not share mother brain completion (although since it syncs the other bosses as well as Zebes wakeup, that last one is kind of an oversight!)

Perhaps you are looking for http://multitroid.com/ ? It does the same thing, but is both more featureful and more actively maintained.

**Comment by GameCrazyMatt** [comment]
Yes! We decided to push forward to see if that was the case late last night. Sorry I didn't post a resolution sooner! Yeah we'll give multitroid a go soon. Thank you!

Sent from Yahoo Mail on Android 
 
  On Sat, Dec 17, 2022 at 1:24 PM, ***@***.***> wrote:   


The emu-coop sharing code for Super Metroid does not share damage, and does not share mother brain completion (although since it syncs the other bosses as well as Zebes wakeup, that last one is kind of an oversight!)

Perhaps you are looking for http://multitroid.com/ ? It does the same thing, but is both more featureful and more actively maintained.
